cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Actions.hs

```diff
@@ -321,10 +322,10 @@ pGovernanceActionTreasuryWithdrawalCmd era = do
             TreasuryWithdrawalCmd
               <$> pNetwork
               <*> pGovActionDeposit
-              <*> pAnyStakeIdentifier
+              <*> pAnyStakeIdentifier Nothing
               <*> pProposalUrl
               <*> pProposalHashSource
-              <*> many ((,) <$> pAnyStakeIdentifier <*> pTransferAmt)
+              <*> many ((,) <$> pAnyStakeIdentifier Nothing <*> pTransferAmt) -- TODO we should likely pass a prefix here, becaus pAnyStakeIdentiefier is used twice
```

Comment:
I agree, I think this should be included in this pr.

Reply:
I would do a separate PR, because it's a different command. I can do this other PR right soon after this one is merged.